src/pages/Dashboard.vue

```diff
@@ -75,16 +76,16 @@ export default defineComponent({
   },
   computed: {
     farmedTotalEarned(): number {
-      if (!this.global.client) return 0
-      return this.global.client.data.farming.farmed.reduce((agg, val) => {
+      if (!this.client) return 0
+      return this.client.data.farming.farmed.reduce((agg, val) => {
         return val.blockReward + val.feeReward + agg
       }, 0)
     },
   },
   unmounted() {
     this.unsubscribe()
-    this.global.client?.do?.blockSubscription.stop()
-    this.global.client?.data?.farming.events.off("farmedBlock", this.farmBlock)
+    this.client.do?.blockSubscription.stop()
+    this.client.data?.farming.events.off("farmedBlock", this.farmBlock)
```

Comment:
Both `this.client.do` and `this.client.data` always exist, remove `?` to prevent confusion.